dnanexus/src/eggd_athena.sh

```
@@ -26,11 +26,19 @@ _set_string_inputs() {
 
     If not specified will be unset, this allows them to be implicitly skipped.
     '''
-    [ -n "$panel" ] && panel="--panel '${panel}' " || unset panel
+    [ -n "$panel" ] && panel_arg="--panel '${panel}' " || unset panel
```

Shellcheck (0.10.0)

[warning] panel_arg appears unused. Verify use (or export if used externally).

(SC2034)